client/signup/config/steps-pure.js

```diff
@@ -737,8 +737,8 @@ export function generateSteps( {
 			apiRequestFunction: setDesignOnSite,
 			delayApiRequestUntilComplete: true,
 			dependencies: [ 'siteSlug' ],
-			providesDependencies: [ 'selectedDesign' ],
-			optionalDependencies: [ 'selectedDesign' ],
+			providesDependencies: [ 'selectedDesign','selectedSiteCategory' ],
```

Comment:
```suggestion
			providesDependencies: [ 'selectedDesign', 'selectedSiteCategory' ],
```
Looks like there are spaces missing after the commas in this file, perhaps that's the reason the Code Style test fails. 🙂